### ISSUE TITLE
percentage count doesn't get color coded, in firefox

### ISSUE THREAD

**Issue by HarshRaj-I** [elliotwaite/thumbnail-rating-bar-for-youtube#63]
example : https://ibb.co/0sP5WFN

please fix this i will be very thankful to you

**Comment by elliotwaite** [comment]
Strange, it seems to be working for me in Firefox. Can you try inspecting the text percentage element to see what the HTML looks like (right-click on the percentage text and choose "Inspect"). For me the element looks like this:

```
<span class="style-scope ytd-video-meta-block ytrb-percentage" style="color:rgb(33.12917594654782,216.75,0)">97.4%</span>
```

And the `style` attribute should be applying the color. Does it look similar for you?

**Comment by elliotwaite** [comment]
After discussing the issue with @HarshRaj-I on Discord, it looks like this issue was being caused by another extension overriding the element's color style by using the `!important` property, so I released an update (version 1.7.1) that fixes this issue by updating the HTML that gets injected by this extension to use the `!import` property as well.
